co2eq/purchase/index.js

```diff
@@ -10,10 +10,18 @@ import {
   UNIT_ITEM,
   UNIT_LITER,
 } from '../../definitions';
-import { convertToEuro, getAvailableCurrencies } from '../../integrations/utils/currency/currency';
 import { getChecksum } from '../utils';
 import footprints from './footprints.yml';
 import consumerPriceIndex from './consumerpriceindices.yml';
+import exchangeRates2011 from './exchange_rates_2011.json';
+
+export function convertTo2011Euro(amount, currency) {
+  if (!Object.keys(exchangeRates2011.rates).includes(currency.toUpperCase())) {
+    throw new Error(`Unknown currency '${currency}'`);
+  }
+  const exchangeRate2011 = exchangeRates2011.rates[currency.toUpperCase()];
```

Comment:
It's better if this line is defined as the first line of the function, to avoid having `toUpperCase()` several places, which can lead to future errors if it's not updated consistently everywhere.
If the key is non existent, then `exchangeRate2011` will simply be `undefined`

Reply:
Right, I thought it would throw an exception